test/encoder.js

```diff
@@ -1,5 +1,6 @@
 import { assert } from 'chai';
 import BN from 'bn.js';
+import BigNumber from 'bignumber.js';
```

Comment:
when will we use BigNumber and when for BN?

Reply:
I needed to switch to `BigNumber.js` because there are a parsing problem with hex strings with `BN.js`. I would suggest to use `BigNumber.js` to keep it uniformly throughout the stacks. But these encoding methods can take both forms.

Reply:
@dwalintukan ic. BN is introduced by ethabijs if I recall correctly. should we remove `import BN from 'bn.js';`?

Reply:
Oh I am testing the encoder method with both BN and BigNumber instances so need the import. Btw, `web3-utils` comes packaged with BN.js.